README.md

```diff
@@ -24,3 +24,21 @@ translators in seconds.
 Zanata is Free software, licensed under the [LGPL][].
 
 [LGPL]: http://www.gnu.org/licenses/lgpl-2.1.html
+
+Developers
+----------
+
+You can use one of the scripts to build the project using maven:
+
+[`etc/scripts/quickbuild.sh`](etc/scripts/quickbuild.sh) - Builds the project as quickly as possible, targeting all
+browsers in the GWT components, and skipping all checks and verifications (i.e.
```

Comment:
If it is going to target all GWT permutations by default, I don't really think you should call it quick.